test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -71,10 +71,10 @@ const mockcheckEntityRequiresTrusts = checkEntityRequiresTrusts as jest.Mock;
 
 const mockGetTrustLandingUrl = getTrustLandingUrl as jest.Mock;
 
-const NEXT_PAGE_URL = "/NEXT_PAGE";
+const MOCKED_URL = "MOCKED_URL/";
```

Comment:
Might be fine, but looks odd (or different to other implementations) - should / be at the end here?

Reply:
i guess it doesn't need it but when used to build the urls the slash was missing if you viewed the url in the html

Reply:
updated